packages/core/nuxt-theme-module/theme/lang/en.js

```diff
@@ -120,6 +120,8 @@ export default {
   'Product suggestions': 'Product suggestions',
   'Search for items': 'Search for items',
   'Enter promo code': 'Enter promo code',
+  'This promo code is invalid': 'This promo code is invalid',
```

Comment:
You are missing translation in german. Please add it as well :)

Reply:
[Added](https://github.com/vuestorefront/vue-storefront/pull/6149/commits/f73929a75dc726802ba4c51122a1920e8b2959fa) ✔️